spec/support/factory_girl.rb

```diff
@@ -14,16 +14,23 @@
 
   factory :game do
     sequence(:name) { |n| "This is a game #{n}" }
-    sequence(:description) { |n| "Description #{n}" }
+    sequence(:description) { |n| "Butcher McSweeney's High Life, PBR&B +1
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.